### DISCUSSION TITLE
Problem with MySQL caching_sha2_password

### DISCUSSION THREAD

**Discussion by guiassemany** [getredash/redash#7066] [Q&A]
Hello, I am trying to connect to a MySQL database that uses the caching_sha2_password password mode. Whenever I test the connection, I get the following error:

```
Connection Test Failed:
(2006, 'RSA Encryption not supported - caching_sha2_password plugin was built with GnuTLS support')
```
I run the latest version of Redash (v10 - redash/redash:10.1.0.b50633) on Ubuntu 18.04 as the main server, and I use Docker containers to serve the app.

Has anyone faced this problem before?

**Comment by d3ividdy** [comment]
hi, change user mysql conf, in database, or create a new user with *mysql_native_password*:

```sql
ALTER USER 'username'@'host' IDENTIFIED WITH mysql_native_password BY 'password';
FLUSH PRIVILEGES;
```
